packages/devtools_app/lib/src/service/vm_service_wrapper.dart

```diff
@@ -77,9 +77,9 @@ class VmServiceWrapper implements VmService {
   /// finish before completing.
   Future<void> forEachIsolate(
       Future<void> Function(IsolateRef) callback) async {
-    final vm = await _vmService.getVM();
+    final vm = await _vmService!.getVM();
     final futures = <Future>[];
-    for (final isolate in vm.isolates) {
+    for (final isolate in vm.isolates!) {
```

Comment:
write `vm.isolates ?? <IsolateRef>[]` instead

Reply:
nit: `?? []` should be sufficient.

Reply:
done